### PR TITLE
#1631 Fix EKM test fail

### LINKED ISSUE #1631: CheckRefreshKeyFromEkmErrorHandling test periodically fails on Semaphore
`CheckRefreshKeyFromEkmErrorHandling` test passes well locally, but sometimes fails on Semaphore:
- https://flowcrypt.semaphoreci.com/jobs/ae259d77-5b58-45dd-8eb2-70e410ea795d
- https://flowcrypt.semaphoreci.com/jobs/93a31342-46f9-43b4-8eb6-4662f57b1c14

Probably related to app restart during test.

### PR DESCRIPTION
**Pull request by sosnovsky**
This PR should fix `CheckRefreshKeyFromEkmErrorHandling` test on Semaphore (I split it into 2 separate tests)

close #1631

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: appium/tests/specs/mock/setup/CheckRefreshKeyFromInvalidKey.spec.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  SplashScreen,
+} from '../../../screenobjects/all-screens';
+import { ekmKeySamples } from "../../../../api-mocks/apis/ekm/ekm-endpoints";
+import { CommonData } from "../../../data";
+import BaseScreen from "../../../screenobjects/base.screen";
+import AppiumHelper from "../../../helpers/AppiumHelper";
+
+describe('SETUP: ', () => {
+
+  it('EKM invalid key update error handled gracefully', async () => {
```
Comment (ioanmo226) [inline appium/tests/specs/mock/setup/CheckRefreshKeyFromInvalidKey.spec.ts:14]: So you tried to make specific test file for this test?

File: FlowCrypt/Controllers/Inbox/InboxViewController.swift [hunk 2]
```diff
@@ -81,31 +85,27 @@ class InboxViewController: ViewController {
...
 
-        tableNode.frame = node.bounds
-
-        if isSearch { return }
+        guard !didLayoutSubviews else { return }
 
-        let offset: CGFloat = 16
-        let size = CGSize(width: 50, height: 50)
+        setupElements()
+        ekmVcHelper.refreshKeysFromEKMIfNeeded(in: self)
```
Comment (ioanmo226) [inline FlowCrypt/Controllers/Inbox/InboxViewController.swift:91]: I wonder why this logic moved to InboxViewController.
And I think with this change, refresh logic will run every time I switch between inbox folders.

Comment (ioanmo226) [reply FlowCrypt/Controllers/Inbox/InboxViewController.swift]: ok

File: appium/tests/specs/mock/setup/CheckRefreshKeyFromInvalidKey.spec.ts (hunk 1 above)
Comment (sosnovsky) [reply appium/tests/specs/mock/setup/CheckRefreshKeyFromInvalidKey.spec.ts:14]: I haven't found exact reason for fail, but when it fails - it happens on 3rd app restart.
Maybe app isn't fully unloaded in this case and it leads to app crash. After test split it passes well each time

Comment (ioanmo226) [reply appium/tests/specs/mock/setup/CheckRefreshKeyFromInvalidKey.spec.ts:14]: OK, got it

**Review by ioanmo226 — APPROVED** [review]

**Pull request merged**